app/bundles/LeadBundle/EventListener/CampaignSubscriber.php

```diff
@@ -51,19 +52,26 @@ class CampaignSubscriber extends CommonSubscriber
      */
     protected $listModel;
 
+    /**
+     * @var CompanyModel
+     */
+    protected $companyModel;
+
     /**
      * CampaignSubscriber constructor.
      *
      * @param IpLookupHelper $ipLookupHelper
      * @param LeadModel      $leadModel
      * @param FieldModel     $leadFieldModel
+     * @param CompanyModel   $companyModel
      */
-    public function __construct(IpLookupHelper $ipLookupHelper, LeadModel $leadModel, FieldModel $leadFieldModel, ListModel $listModel)
+    public function __construct(IpLookupHelper $ipLookupHelper, LeadModel $leadModel, FieldModel $leadFieldModel, ListModel $listModel, CompanyModel $companyModel)
     {
-        $this->ipLookupHelper = $ipLookupHelper;
-        $this->leadModel      = $leadModel;
-        $this->leadFieldModel = $leadFieldModel;
-        $this->listModel      = $listModel;
+        $this->ipLookupHelper    = $ipLookupHelper;
+        $this->leadModel         = $leadModel;
+        $this->leadFieldModel    = $leadFieldModel;
+        $this->listModel         = $listModel;
+        $this->companyModel      = $companyModel;
```

Comment:
Was the indentation change for all properties in the constructor required? I don't see a reason. Please change it back.